.github/workflows/ci.yml

```diff
@@ -11,5 +11,25 @@ on:
       - 'master'
 
 jobs:
+  deploy-clojars:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Get an env
+        uses: yetanalytics/actions/setup-env@v0
+      
+      - name: Test deployment
+        uses: yetanalytics/actions/deploy-clojars@clojars
+        with:
+          artifact-id: 'actions'
+          version: 'LATEST'
+          resource-dirs: '[]'
+          publish: false
```

Comment:
Cool! So this is basically a dry run? Great design

Reply:
Yeah I quickly realized that commenting and uncommenting the deploy step was not a sustainable long-term solution.